### ISSUE TITLE
better CONTRIBUTING.md

### ISSUE THREAD

**Issue by beatrix-bitrot** [glitch-soc/mastodon#48]
the existing one is kinda terse/minimal/code focused and only has links to tootsuite

i reckon we can make it nicer and better

**Comment by marrus-sh** [comment]
definitely, I think that different roles (mgmt/design/code/testing/docs) should be better laid out and explained, and also just like, a "here is how our project hierarchy is set up," "here's where you go to get feedback on stuff," "this is what you can expect from Pull Requests," "this is what you can expect when you submit an issue," and so forth.

this might be good information to put in the documentation repo too if we want to leave `CONTRIBUTING.md` more code-based.
